docs/docs/installation_guide/installation/pre-installation/oidc-providers.md

```diff
@@ -0,0 +1,127 @@
+---
+id: oidc-providers
+title: OIDC Providers Settings
+sidebar_label: OIDC Providers Settings
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+This section is for creating OIDC IdP client and creating Service Workbench root user in OIDC IdP.
+
+## Creating OIDC IdP Client
+
+[Cognito User Pool][cognito] is not supported in AWS China Regions. If you choose to deploy the solution in AWS China Regions, an OpenID Connector (OIDC) IdP client is required.
```

Comment:
OpenID Connect (OIDC) IdP is an alternative of Cognito User Pool for authentication of Service Workbench. Before deploying Service Workbench with OIDC IdP, you have to create your OIDC provider in external OIDC services. 

Below are example procedures for creating OIDC providers in Authing, Keycloak and Okta for reference. You can use other OIDC providers as well.

Reply:
Refactored